File: javascript/1150.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split("\n");
var a = parseInt(lines.shift());
var sum = a;
var count = 1;

while(lines.length){
    var b = parseInt(lines.shift());

    if(b > a) break;
}

while(sum < b){
    sum += a + count;
    count++;
}

console.log(count);

File: python/1150.py
count = 1
a = int(input())
sm = a

while True:
    try:
        b = int(input())
        if b > a:
            break
    except EOFERROR:
        break
    
while sm < b:
    sm += a + count
    count += + 1
    
print(count)